platformics/api/core/gql_loaders.py

```diff
@@ -105,7 +106,7 @@ def group_by_remote_key(row: Any) -> Tuple:
             self._loaders[(relationship, input_hash)] = DataLoader(load_fn=load_fn)  # type: ignore
             return self._loaders[(relationship, input_hash)]  # type: ignore
         
-    def aggregate_loader_for(self, relationship: RelationshipProperty, where: Optional[Any] = None, aggregate: Optional[Any] = None) -> DataLoader:
+    def aggregate_loader_for(self, relationship: RelationshipProperty, where: Optional[Any] = None, selections: Optional[Any] = None) -> DataLoader:
```

Comment:
I think (?) our `input_hash` will need to include our aggregate fields and our group by selections when computing the hash.